app/src/app/utils/events/mapEvents.ts

```diff
@@ -145,7 +145,7 @@ export const handleMapMouseMove = (
 export const handleMapZoom = (
   e: MapLayerMouseEvent | MapLayerTouchEvent,
   map: MutableRefObject<Map | null>,
-  hoverFeatureIds: React.MutableRefObject<Set<string>>
+  hoverFeatureIds: React.MutableRefObject<Set<string>>,
```

Comment:
O: We should align on formatters! I think you're using the same one as @mariogiampieri , VS Code's prettier?